src/main/java/com/google/devtools/build/lib/remote/RemoteSpawnRunner.java

```diff
@@ -207,7 +207,6 @@ public SpawnResult exec(Spawn spawn, SpawnExecutionContext context)
     boolean uploadLocalResults = remoteOptions.remoteUploadLocalResults && spawnCacheableRemotely;
     boolean acceptCachedResult = remoteOptions.remoteAcceptCached && spawnCacheableRemotely;
 
-    context.report(ProgressStatus.EXECUTING, getName());
```

Comment:
Maybe change this to PREPARING or similar so that the action shows up in the UI?

Reply:
Good point. I will try this.

Reply:
New commit changes this to SCHEDULING so actions waiting for execution are displayed as `[Sched] ...` rather than `[Prepa]`.